pkg/ride/converters.go

```diff
@@ -1347,13 +1356,20 @@ func signatureToProofs(sig *crypto.Signature) RideList {
 
 func proofs(proofs *proto.ProofsV1) RideList {
 	r := make(RideList, 8)
```

Comment:
if you don't to `make(RideList, 8)` for just 8 cells, there will be no need to do this
```
else {
		// special case for ethereum stuff
		for i := range r {
			r[i] = RideBytes(nil)
		}
	}
```

Reply:
Is there any specific point to do `every r[i] = RideBytes(nil)` instead of just `r = nil`?

Reply:
In short, yes. We should set empty value for each element of `RideList` according to the specification.